invenio/ext/login/__init__.py

```diff
@@ -126,10 +130,8 @@ def authenticate(nickname_or_email=None, password=None,
         return False
 
     if user.note == '2':  # account is not confirmed
-        logout_user()
```

Comment:
Note: This statement I don't think have had any effect since control flow continues down to the return statement below (``return login_user(..)``). Probably we should have a configuration variable to decide if a user can login without email confirmation, but I think that requires decomposing ``user.note`` into separate fields (e.g. active and confirmed booleans).